Tests/ArcGISToolkitTests/FeatureTemplatePickerTests.swift

```diff
@@ -0,0 +1,95 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+@testable import ArcGISToolkit
+import Foundation
+import os
+import Testing
+
+@Suite("FeatureTemplatePicker Tests")
+struct FeatureTemplatePickerTests {
+    @Test
+    @MainActor
+    func testGeoModelWithTemplates() async throws {
+        let map = makeMapWithTemplates()
+        try await map.load()
+        
+        let model = FeatureTemplatePicker.Model(geoModel: map, includeNonCreatableFeatureTemplates: true)
+        #expect(model.includeNonCreatableFeatureTemplates)
+        #expect(!model.isGeneratingFeatureTemplates)
+        #expect(!model.showContentUnavailable)
+        #expect(!model.showNoTemplatesFound)
+        
+        let lockedValues = OSAllocatedUnfairLock(initialState: Array<Bool>())
+        let subscription = model.$isGeneratingFeatureTemplates.sink { isGeneratingFeatureTemplates in
```

Comment:
<img width="1130" alt="Screenshot 2025-01-03 at 1 23 26 PM" src="https://github.com/user-attachments/assets/e0ce7c38-9109-453d-a0dc-c68d2545a584" />

Unfortunately, this property and the same property in the test below generate a warning, and applying the suggestion causes the test to fail.

How about calling `subscription.cancel()` at the end of each test?